spec/fixtures/transfers/201_transfer_create.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
As mentioned in the PR, we should progressively move to use HTTP files and webmock.